oper8/deploy_manager/openshift_deploy_manager.py

```diff
@@ -680,6 +692,20 @@ def _manifest_diff(cls, manifest_a, manifest_b):
         log.debug3("B: %s", manifest_b)
         return change
 
+    @classmethod
+    def _requires_replace(cls, manifest_a, manifest_b) -> bool:
+        """Helper to compare two manifests to see if they require
+        replace
+
+        Returns:
+            bool: If the resource requires a replace operation
+        """
+
+        manifest_a, manifest_b = cls._clean_manifest(manifest_a, manifest_b)
+
+        change = bool(requires_replace(manifest_a, manifest_b))
+        log.debug2("Requires Replace? %s", change)
+
```

Comment:
It seems we need to return `change` here.
```suggestion
        return change
```